### DISCUSSION TITLE
No way to remove Action buttons (share, like) without removing controls(play), No way to remove end screen recommendations.

### DISCUSSION THREAD

**Discussion by saurabhverma2892** [vimeo/player.js#991] [Q&A]
### Expected Feature
I could find no options to remove the like, share action buttons without the play controls getting hidden too.
Also, Id like the player to not show other video recommendations at the end.
Using a premium account.

### Actual Behavior
would be nice to have it in options to remove the action buttons and end screen recommendations

**Comment by TomHell** [comment]
Any decission or update regarding this request?
Thanks
